docs/documentation/publishing-on-govuk-paas.md

```diff
@@ -0,0 +1,93 @@
+# Publishing on the web (GOV.UK PaaS)
+
+We recommend using [GOV.UK PaaS](https://www.cloud.service.gov.uk) to get your prototype online.  It’s simple and fast to deploy new versions as you work.
+
+Once your prototype is on GOV.UK PaaS, other people will be able to access and try your prototype from their own computers or mobile devices.
+
+> **DO NOT** enter real user data in to prototypes hosted on GOV.UK PaaS. If your prototype stores or collects user data, talk to a security professional about appropriate security steps you must take.
+
+A prototype deployed to GOV.UK PaaS is called an `app` - it will have a url like:
+
+`your-prototype.london.cloudapps.digital`
+
+or
+
+`your-prototype.cloudapps.digital`
+
+depending on which region your GOV.UK PaaS account is in.
+
+You can have multiple apps running on GOV.UK PaaS - projects often have several so they can try different ideas out at once.
+
+## 1) Get started with GOV.UK PaaS
+
+Follow the [GOV.UK PaaS gettin started guide](https://docs.cloud.service.gov.uk/get_started.html#get-started) to:
```

Comment:
🤦‍♀️
```suggestion
Follow the [GOV.UK PaaS getting started guide](https://docs.cloud.service.gov.uk/get_started.html#get-started) to:
```

Reply:
Resolved in 7575565b6d2b9f245bba97ec9c2be83cf005cd34 with the actual page title :D